src/Microsoft.VisualStudio.LanguageServices.Razor/RazorEditorParser.cs

```diff
@@ -1,190 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System;
-using System.Diagnostics;
-using Microsoft.AspNetCore.Razor.Language;
-using Microsoft.AspNetCore.Razor.Language.Legacy;
-using Microsoft.VisualStudio.Text;
-
-namespace Microsoft.VisualStudio.LanguageServices.Razor
-{
-    public class RazorEditorParser : IDisposable
```

Comment:
This was moved into the Legacy folder and the BackgroundParser for it was added as a private class (why git doesn't see it as moved)